santoku/aws/sqs.py

```diff
@@ -96,12 +95,9 @@ def get_queue_url(self, queue_name: str) -> str:
         https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-general-identifiers.html
 
         """
-        try:
```

Comment:
why removing the try/catch?

Reply:
In all the try/catch we removed we were not doing anything with the error that we expected, we just raised the same error because the error raised by botocore were already sufficiently clearly explained.